storage/Cargo.toml

```diff
@@ -20,10 +20,13 @@ num_cpus = "1.13"
 rocksdb = {version = "0.15", features = ["snappy", "lz4", "zstd", "zlib"], default-features = false }
 serde = { version = "1.0", features = ["derive", "rc"] }
 slog = { version = "2.7", features = ["max_level_trace", "release_max_level_debug"] }
-sled = "0.34.6"
+sled = {version = "0.34.6", features=["compression"]}
 snap = "1.0.4"
 strum = "0.20"
 strum_macros = "0.20"
+bloomfilter = "1.0.5"
+notus = { git = "https://github.com/mambisi/notus.git"}
```

Comment:
@mambisi @jurajselep 
what about this?
if we use git reference, it should better point to some fixed value, like tag or commit_hash,
anyway, `notus` should be migrated under tezege, right?
`https://github.com/tezedge/notus.git`